components/header.js

```diff
@@ -41,10 +35,64 @@ const Header = ({ loading }) => {
             menuItems={[
               <Dimmer key='dimmer' sx={{ mt: '-2px', color: 'primary' }} />,
             ]}
-            status={status}
           />
         </Container>
       </Box>
+      <Box
+        sx={{
+          pointerEvents: 'none',
+          position: 'absolute',
+          left: [
+            `calc(3 * 100vw / 6 - 12px)`,
```

Comment:
Hm it strikes me that we're positioning the spinner and the "Release to update" text relative to the `ControlPanel`. Perhaps these should just be props we pass in. If that feels good to you, I'm happy to refactor as part of the `SidebarPanel` -> `layouts` change.

Reply:
yep that was the goal, happy with whatever you want to do there! it's interesting because layout-wise it's similar to the arrow in the bottom left (positioned relative to the panel), but functionally it's fairly separate... maybe you want a `SidebarAttachment` (or some better name?) to describe something adjacent to the sidebar, whereas (presumably?) the arrow is going to be built into the `SidebarPanel`?